openedx/core/djangoapps/programs/management/commands/backpopulate_program_credentials.py

```diff
@@ -111,5 +111,6 @@ def _load_usernames(self):
 
         query = status_query & course_run_query
 
-        username_dicts = GeneratedCertificate.eligible_certificates.filter(query).values('user__username').distinct()
+        username_dicts = \
```

Comment:
I believe the slash for a new line is discouraged when there are other options in the code style guide we use and PEP8, can we update this line to not need the \ ?

Reply:
done